great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -474,6 +502,10 @@ def save_datasource(
         self._cached_datasources[datasource_name] = updated_datasource
         return updated_datasource
 
+    @usage_statistics_enabled_method(
+        event_name=UsageStatsEvents.DATA_CONTEXT_ADD_DATASOURCE.value,
```

Comment:
Can we use the `str` mixin to get rid of these `value` calls?

Reply:
for sure! mind if I do that in a subsequent PR? The aim with this PR is to simply port over methods as-is

Reply:
Works for me!